frontend/src/routes/candidate/(protected)/profile/+page.svelte

```diff
@@ -79,7 +79,7 @@
       <label for="portrait" class={labelClass}>
         {$_('candidateApp.basicInfo.fields.unaffiliated')}
       </label>
-      <input type="checkbox" class="toggle toggle-primary mr-8" checked />
+      <input type="checkbox" class="toggle-primary toggle mr-8" checked />
```

Comment:
For some reason prettier formats these differently on different devices. I and Stepan think they were wrong in this PR but since it was already merged, I suppose we will fix it later.